src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -358,17 +359,46 @@ def help_text
       #
       # @return [Boolean] true when skipping has been confirmed
       def show_skipping_warning
+        media_name = ProductFeatures.GetStringFeature(
+          "globals",
+          "full_system_media_name"
+        )
+        download_url = ProductFeatures.GetStringFeature(
+          "globals",
+          "full_system_download_url"
+        )
+
+        warning = _("Without registration, update channels will not be\n" \
+          "configured. This disables updates and security fixes.")
+
         # Popup question: confirm skipping the registration
         # TRANSLATORS:
         # %{media_name} is the media name (e.g. SLE-15-Packages),
         # %{download_url} is an URL link (e.g. https://download.suse.com)
-        warning = _("Without registration, update channels will not be\n" \
-          "configured. This disables updates and security fixes.\n\n" \
-          "A full system can be installed using the\n" \
-          "%{media_name} media from %{download_url}.\n" \
-          "Without these media only a minimum system is available\n" \
-          "in this installation.") %
-          { media_name: "SLE-15-Packages", download_url: "https://download.suse.com" }
+        if media_name && !media_name.empty? &&
```

Comment:
The `nil` check is actually not needed, `ProductFeatures.GetStringFeature` returns empty string if it is not found, see [here](https://github.com/yast/yast-yast2/blob/5b9a565af09703f569cff96e7f1a96a25b4cc903/library/control/src/modules/ProductFeatures.rb#L215). Just remove the `media_name && download_url` checks and add a comment describing that so it is clear.